services/payment/src/EShopOnAbp.PaymentService.Application.Contracts/PaymentMethods/PaymentTypeIds.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EShopOnAbp.PaymentService.PaymentMethods;
+
+public static class PaymentTypeIds
```

Comment:
I think we can merge and group PaymentTypeIds and PaymentType strings. Like permission groups.